source/guides/stacks/mers.rst

```diff
@@ -108,17 +149,17 @@ Getting the pre-built |MERS| container image
 ********************************************
```

Comment:
General rule, don't use -ing words in section titles. Instead "Get the pre-built MeRS container image"